item_selection/book recommenders/goodreads_1k_data.csv

```diff
@@ -0,0 +1,1005 @@
+"Strong Motion. The critically acclaimed second novel from the author of 'The Corrections'. 'Strong Motion' is the brilliant, bold second novel from the bestselling and critically acclaimed author of 'The Corrections' and 'Freedom'. Louis Holland arrives in Boston in a spring of strange happenings - earthquakes strike the city, and the first one kills his grandmother. During a bitter feud over the inheritance Louis falls in love with Renee Seitchek, a passionate and brilliant seismologist, whose discoveries about the origin of the earthquakes complicate everything. Potent and vivid, 'Strong Motion' is a complex story of change from the forceful imagination of Jonathan Franzen.","The Invisible Man. This masterpiece of science fiction is the fascinating story of Griffin, a scientist who creates a serum to render himself invisible, and his descent into madness that follows.",Setting Free the Bears / The Water-Method Man / The 158-Pound Marriage.,"Accordion Crimes. Pulitzer Prize-winning author Annie Proulx brings the immigrant experience to life in this stunning novel that traces the ownership of a simple green accordion.
```

Comment:
Having data structured as having separate column for each ID instead of row is a bit awkward. I assume it's to be consistent with Selective format?

Reply:
Yes, this is consistent with Selective format, where each column is treated as a feature in other selection methods I think.